docs/content/dependencies/deep-storage.md

```diff
@@ -19,26 +21,6 @@ Note that you should generally set `druid.storage.storageDirectory` to something
 
 If you are using the Hadoop indexer in local mode, then just give it a local file as your output directory and it will work.
 
-## S3-compatible
```

Comment:
I think it'd be good to leave the header but then in the body of this section, deep link to the relevant extension doc. That way it's clear that we support HDFS/S3 and there are not too many clicks needed to get there.